tests/sitl/test_localposition.py

```diff
@@ -0,0 +1,22 @@
+from dronekit import connect
+from dronekit.tools import with_sitl
+import time
+from nose.tools import assert_not_equals
+
+@with_sitl
+def test_timeout(connpath):
+    v = connect(connpath, await_params=True)
+
+    v.armed = True
+    v.flush()
+
+    time.sleep(5)
```

Comment:
@tcr3dr 5 more secs on our tests, have you played with this number before to try to tune it?